setup.py

```diff
@@ -25,7 +25,12 @@
     author_email='benedict@soe.usc.edu',
     url="https://github.com/BD2KGenomics/toil",
     install_requires=[
-        'bd2k-python-lib==1.13.dev14'],
+        'bd2k-python-lib==1.13.dev14',
+        'dill==0.2.5'],
+    tests_require=[
```

Comment:
BTW, I didn't catch this earlier but this is a merge error on your part, @jpfeil. You reintroduced lines that I had taken out. Try to be extra careful with merges!